src/builder/Actions.sol

```diff
@@ -1021,8 +1211,8 @@ library Actions {
         return ds[0];
     }
 
-    function emptyRepayActionContext() external pure returns (RepayActionContext memory) {
-        RepayActionContext[] memory rs = new RepayActionContext[](1);
+    function emptyMorphoRepayActionContext() external pure returns (MorphoRepayActionContext memory) {
+        MorphoRepayActionContext[] memory rs = new MorphoRepayActionContext[](1);
         return rs[0];
     }
```

Comment:
oops my bad on "updating" this line instead of add `emptyMorphoreapyActionContext()` instead 😓 

And yeah if I understand it correctly this are not meant to be used in builder code. It's for client it seems `// These empty husk functions exist so that the structs make it into the abi so the clients can know how to decode them.`